aws/service_v2.go

```diff
@@ -81,6 +82,14 @@ func CostExplorerClient(ctx context.Context, d *plugin.QueryData) (*costexplorer
 	return costexplorer.NewFromConfig(*cfg), nil
 }
 
+func DocDBService(ctx context.Context, d *plugin.QueryData) (*docdb.Client, error) {
```

Comment:
```suggestion
func DocDBClient(ctx context.Context, d *plugin.QueryData) (*docdb.Client, error) {
```

This function name should be consistent with the others